### PR TITLE
Implement toolkit csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Migrate Toolkit Sample -**

1 . tab-personal mvc
2 . tab-personal razor
3 . tab-people picker csharp
4 . tab-channel-group mvc-csharp

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline  -  
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7113491566
image

**Comment by Harikrishnan-MSFT** [comment]
**Migrate Toolkit Sample -** 

1 . tab-personal mvc 
2 . tab-personal razor 
3 . tab-people picker csharp 
4 . tab-channel-group mvc-csharp  csharp

File: samples/tab-channel-group/mvc-csharp/README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
   
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
```
Comment (formulahendry) [inline samples/tab-channel-group/mvc-csharp/README.md:30]: Add `Teams Toolkit` into Prerequisites. Also for other README.

File: samples/tab-channel-group/mvc-csharp/README.md [hunk 2]
```diff
@@ -29,6 +29,17 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
...
 
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.8 Preview 4 or higher** [Visual Studio](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#1790-pre10--visual-studio-2022-version-179-preview-1)
```
Comment (formulahendry) [inline samples/tab-channel-group/mvc-csharp/README.md:35]: There is stable Version 17.8 now. So we could just say **Version 17.8 or higher**. . Also for other README.

File: samples/tab-personal/mvc-csharp/teamsapp.local.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:tab-personal-csharp
```
Comment (formulahendry) [inline samples/tab-personal/mvc-csharp/teamsapp.local.yml:7]: tab-personal-mvc-csharp

File: samples/tab-personal/mvc-csharp/teamsapp.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:tab-personal-csharp
```
Comment (formulahendry) [inline samples/tab-personal/mvc-csharp/teamsapp.yml:7]: tab-personal-mvc-csharp

File: samples/tab-personal/razor-csharp/teamsapp.local.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:tab-personal-csharp
```
Comment (formulahendry) [inline samples/tab-personal/razor-csharp/teamsapp.local.yml:7]: tab-personal-razor-csharp

File: samples/tab-personal/razor-csharp/teamsapp.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:tab-personal-csharp
```
Comment (formulahendry) [inline samples/tab-personal/razor-csharp/teamsapp.yml:7]: tab-personal-mvc-csharp

Comment (Harikrishnan-MSFT) [reply samples/tab-personal/razor-csharp/teamsapp.yml:7]: Fixed

File: samples/tab-personal/razor-csharp/teamsapp.local.yml (hunk 5 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-personal/razor-csharp/teamsapp.local.yml:7]: Fixed

File: samples/tab-personal/mvc-csharp/teamsapp.yml (hunk 4 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-personal/mvc-csharp/teamsapp.yml:7]: Fixed

File: samples/tab-personal/mvc-csharp/teamsapp.local.yml (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-personal/mvc-csharp/teamsapp.local.yml:7]: Fixed

File: samples/tab-channel-group/mvc-csharp/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group/mvc-csharp/README.md:30]: Fixed

File: samples/tab-channel-group/mvc-csharp/README.md (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group/mvc-csharp/README.md]: Fixed

File: samples/tab-channel-group/mvc-csharp/README.md [hunk 7]
```diff
@@ -29,10 +29,12 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
   
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+- Teams Toolkit
+
 ## Run the app (Using Teams Toolkit for Visual Studio)
 
 The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
-1. Install Visual Studio 2022 **Version 17.8 Preview 4 or higher** [Visual Studio](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#1790-pre10--visual-studio-2022-version-179-preview-1)
+1. Install Visual Studio 2022 **Version 17.8 or higher** [Visual Studio](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#1790-pre10--visual-studio-2022-version-179-preview-1)
```
Comment (formulahendry) [inline samples/tab-channel-group/mvc-csharp/README.md:37]: Need to update URL as well.

File: samples/tab-channel-group/mvc-csharp/teamsapp.local.yml [hunk 8]
```diff
@@ -4,7 +4,7 @@
 version: v1.2
 
 additionalMetadata:
-  sampleTag: Microsoft-Teams-Samples:tab-channel-group-csharp
+  sampleTag: tab-channel-group-mvc-csharp
```
Comment (formulahendry) [inline samples/tab-channel-group/mvc-csharp/teamsapp.local.yml:7]: Please DO NOT remove 'Microsoft-Teams-Samples:'!
The format is `<repo_name>:<sampe_name>`

File: samples/tab-channel-group/mvc-csharp/README.md [hunk 9]
```diff
@@ -29,6 +29,19 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
   
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+- Teams Toolkit
```
Comment (formulahendry) [inline samples/tab-channel-group/mvc-csharp/README.md:32]: `Teams Toolkit for Visual Studio`, and also better to add a link.

File: samples/tab-channel-group/mvc-csharp/teamsapp.local.yml (hunk 8 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group/mvc-csharp/teamsapp.local.yml:7]: Fixed

File: samples/tab-channel-group/mvc-csharp/README.md (hunk 9 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-channel-group/mvc-csharp/README.md]: Fixed

**Review by formulahendry — APPROVED** [review]

**Review by summzhan — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7178717485

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, approving!

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, approving!

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7954951294**

**Pull request merged**